.pre-commit-config.yaml

```
@@ -39,6 +39,7 @@ repos:
     hooks:
       - id: "tmt-tests-lint"
         verbose: false
+        files: tests/
```

We should maybe add also `plans` folder
```suggestion
        files: 
          - tests/
          - plans/
```